iphone/cli/commands/_buildModule.js

```diff
@@ -473,6 +473,11 @@ iOSModuleBuilder.prototype.buildModule = function buildModule(next) {
 			args.push('CONFIGURATION_BUILD_DIR=' + path.join(this.projectDir, 'build', 'Release-' + target));
 		}
 
+        // Exclude arm64 architecture from simulator build - TIMOB-28042
```

Comment:
<!--
  1 failure:  iphone/cli/comman...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: iphone/cli/commands/_buildModule.js;
  Line: 476;
-->
- :no_entry_sign: iphone/cli/commands/_buildModule.js line 476 – Expected indentation of 2 tabs but found 8 spaces. (indent)